networkx/algorithms/coloring/greedy_coloring.py

```diff
@@ -202,7 +202,20 @@ def strategy_saturation_largest_first(G, colors):
                 distinct_colors[neighbour].add(color)
 
 
-def greedy_color(G, strategy=strategy_largest_first, interchange=False):
+GREEDY_COLOR_STRATEGIES = {
```

Comment:
How about naming this simply `STRATEGIES`, since it already lives in the module `greedy_coloring`?